extensions/twenty/src/create-object-record-form.tsx

```diff
@@ -0,0 +1,73 @@
+import { Action, ActionPanel, Form, Icon, PopToRootType, Toast, showHUD, showToast } from "@raycast/api";
+import { Fragment } from "react";
+
+import { FailureToast } from "./enum/api";
+
+import { DataModelWithFields, ObjectRecordFields } from "./services/zod/schema/recordFieldSchema";
+import { FieldComponent, TextInput } from "./components";
+import { useForm } from "@raycast/utils";
+import twenty from "./services/TwentySDK";
+import { createValidationsForRest } from "./helper/createValidationsForRest";
+import { formatValues } from "./helper/formatValues";
+
+function CreateObjectRecordForm({
+  objectRecordMetadata,
+  fields,
+}: {
+  objectRecordMetadata: DataModelWithFields;
+  fields: ObjectRecordFields;
+}) {
+  const { namePlural, labelSingular } = objectRecordMetadata;
+  const { primary, rest } = fields;
+  const { handleSubmit, itemProps } = useForm({
+    async onSubmit(values) {
+      await showToast({
+        style: Toast.Style.Animated,
+        title: "Creating Object Record",
+      });
+
+      const formattedValues = formatValues(values, objectRecordMetadata);
+      const isSuccess = await twenty.createObjectRecord(namePlural, formattedValues);
+
+      if (isSuccess) {
+        await showHUD(`${objectRecordMetadata.labelSingular} Created 🎉`, {
+          popToRootType: PopToRootType.Immediate,
+          clearRootSearch: true,
+        });
+
+        return;
+      }
```

Comment:
Could we change this - maybe turn it into a `Preference`?
Once an item is added, I'd either want to enter another item (reset form) or I'd assume the `Form` will go to previous view. The `hud` can stay but I'd want to be able to choose whether to stay or close like it does right now.